docs/aries-endorser-service-alert-workflow.md

```diff
@@ -0,0 +1,39 @@
+# Aries Endorser Service Uptime Alert Workflow
+
+The following workflow describes the steps to confirm and resolve an Aries Endorser Service alert condition on one of the Endorser Service instances.  Addition information regarding the associated error condition and its impact, along with details of the steps to resolve it can be found below.
+
+<p align="center">
+  <img src="https://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/bcgov/DITP-DevOps/main/docs/diagrams/endorser-service-decision-tree.puml">
+</p>
+
+## What does an alert look like?
+
+![aries-endorser-service-alert](./images/aries-endorser-service-alert.png)
+
+## About the Aries Endorser Service
+
+The Aries Endorser Service is an Aries Agent instance complete with a proxy server configured as a endorser for use with mobile wallets such as the BC Wallet.  The service acts as the endorser between the mobile wallet and other Aries Agent instances that are interacting with the mobile wallet to issue or verify credentials.
+
+### What is the impact of the Aries Endorser Service being down?
+
+Any mobile wallet configured to use the service as its endorser ceases to function properly, as it relies on the endorser to communicate with other agents.
```

Comment:
```suggestion
Any author agent configured to use the service as its endorser will be unable to write new transactions to its corresponding ledger, as the author relies on the endorser to sign (endorse) the transactions so they may be written to the ledger.  This can be a serious issue for agents that need to write revocation registry entries to the ledger to revoke credentials.
```